src/sentry/lang/javascript/processor.py

```diff
@@ -160,15 +160,17 @@ def discover_sourcemap(result):
         # are generous and assume it's somewhere either in the first or last 5 lines.
         # If it's somewhere else in the document, you're probably doing it wrong.
         if len(parsed_body) > 10:
-            possibilities = set(parsed_body[:5] + parsed_body[-5:])
+            possibilities = parsed_body[:5] + parsed_body[-5:]
         else:
-            possibilities = set(parsed_body)
+            possibilities = parsed_body
 
+        # We want to scan each line sequentially, and the last one found wins
+        # This behavior is undocumented, but matches what Chrome and Firefox do.
         for line in possibilities:
-            if line.startswith('//@ sourceMappingURL=') or line.startswith('//# sourceMappingURL='):
+            pragma = line[:21]
+            if pragma in ('//# sourceMappingURL=', '//@ sourceMappingURL='):
                 # We want everything AFTER the indicator, which is 21 chars long
                 sourcemap = line[21:].rstrip()
```

Comment:
I think this should technically be `line[21:].strip()` too, to trim spaces at the beginning. Thoughts?